plugins/MCBEEssentials/manifest.json

```diff
@@ -2,15 +2,15 @@
 	"author": "rebrainertv",
 	"icon": "mdi-wrench-outline",
 	"name": "MCBE Essentials",
-	"version": "1.0.0",
+	"version": "1.1.0",
 	"id": "52e9fb51-4733-4336-85f4-94ed94bbb539",
-	"description": "Open the MCBE Essentials website within a new bridge. tab",
+	"description": "This extension connects MCBE Essentials apps with bridge. Right click a trade table, dialogue, structure or recipe file and select Open With to open it in the appropriate MCBE Essentials app! In addition, the site is available as a sidebar button.",
 	"api_version": 2,
 	"target": "v2",
 	"compatibleAppVersions": {
 		"min": "2.3.0"
 	},
-	"readme": "https://mcbe-essentials.github.io/",
+	"readme": "https://mcbe-essentials.glitch.me/",
```

Comment:
Is this extension supposed to still be using the glitch url? There are a few occurrences of this URL throughout the extension

Reply:
It should work either way since a user that hasn't signed up for MCBE Essentials Nightly will automatically get redirected to the stable version of whatever page they're on, but it might be a better idea to change it.